extensions/amp-analytics/0.1/test/test-visibility-impl.js

```diff
@@ -248,6 +248,7 @@ describes.realWin('amp-analytics.visibility', {amp: true}, env => {
           elementY: '75',
           firstSeenTime: '100',
           fistVisibleTime: '100',
```

Comment:
I can't. Test requires it at this time. Will be removed automatically when visibility-v3 is cleaned up.